geomesa-spark/geomesa-spark-sql/src/test/scala/org/locationtech/geomesa/spark/SparkSQLColumnsTest.scala

```diff
@@ -128,17 +129,20 @@ class SparkSQLColumnsTest extends Specification with LazyLogging {
         "boolean" -> DataTypes.BooleanType,
         "dtg"     -> DataTypes.TimestampType,
         "time"    -> DataTypes.TimestampType,
+        "bytes"   -> DataTypes.BinaryType,
+        "list"    -> ArrayType(DataTypes.StringType),
+        "map"     -> MapType(DataTypes.StringType, DataTypes.IntegerType),
         "line"    -> JTSTypes.LineStringTypeInstance,
         "poly"    -> JTSTypes.PolygonTypeInstance,
         "points"  -> JTSTypes.MultiPointTypeInstance,
         "lines"   -> JTSTypes.MultiLineStringTypeInstance,
         "polys"   -> JTSTypes.MultipolygonTypeInstance,
-        "geoms"   -> JTSTypes.GeometryTypeInstance,
```

Comment:
"mixed" i.e. generic Geometry types get mapped to byte arrays iirc. does this change mean that those get confused with actual byte arrays?

Reply:
To my understanding, `Geometry` and `GeometryCollection` types were mapped to JTS UDTs thus these can be distinguished from byte arrays.